src/main/java/Deadline.java

```diff
@@ -0,0 +1,32 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+
+    LocalDate localDate;
+
+    String formattedDescription;
+
+    String dateString;
+
+    Deadline(String description) throws EmptyArgumentException, BadDateException {
+        super(description);
+        int indexOfDate = description.indexOf("/") + 4;
```

Comment:
Might want to declare what the value `4` means? If not will be classified as magic number.